clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts

```diff
@@ -4,18 +4,29 @@
  *--------------------------------------------------------------------------------------------*/
 import { CollectionRequestParams, IModelScopedOperationParams } from "../../base";
 
+/** Special value which is an alias for the current user id. */
 export const SPECIAL_VALUES_ME = "me";
 
+/**
+ * Valid values for Briefcase owner id filter. Currently the only supported value is `me` which allows to query
+ * Briefcases owned by the current user making the request.
+ */
 export type ValidOwnerIdValue = typeof SPECIAL_VALUES_ME;
 
+/** Url parameters supported in Briefcase list query. */
 export interface GetBriefcaseListUrlParams extends CollectionRequestParams {
+  /** Filters Briefcases with specific owner. See {@link ValidOwnerIdValue}. */
   ownerId?: ValidOwnerIdValue;
 }
 
+/** Parameters for get Briefcase list operation. */
 export interface GetBriefcaseListParams extends IModelScopedOperationParams {
+  /** Parameters that will be appended to the entity list request url that will narrow down or alter the results. */
```

Comment:
I think alter the results is not needed, it is a bit misleading in my opinion. We should just leave narrow down the results.

Reply:
Fixed (here and in other places where the same sentence is used): https://github.com/iTwin/imodels-clients/blob/b3a0ba5c80c6c55204fa675ed18de31a703c9e36/clients/imodels-client-management/src/operations/briefcase/BriefcaseOperationParams.ts#L24